<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetShowInfo</class>
 <widget class="QWidget" name="SetShowInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>字体设置</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="verticalSpacing">
       <number>15</number>
      </property>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QRadioButton" name="but_time">
          <property name="text">
           <string>时间</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="but_date">
          <property name="text">
           <string>日期</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0" colspan="2">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>字体</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFontComboBox" name="fontComboBox"/>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>大小</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_size"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>间距：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="horizontalSlider_space">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>背景图片设置</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="2">
       <widget class="QPushButton" name="but_open">
        <property name="text">
         <string>选择自定义图片</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="3">
       <widget class="QLineEdit" name="lineEdit_path"/>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>无</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>:/image/night.jpg</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>:/image/starrySky.jpg</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>颜色设置</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QRadioButton" name="but_font">
        <property name="text">
         <string>字体颜色</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="but_bg">
        <property name="text">
         <string>背景颜色</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="horizontalSpacing">
         <number>10</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>R</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSlider" name="horizontalSlider_R">
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="2" rowspan="4">
         <widget class="QLabel" name="label_color">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>G</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSlider" name="horizontalSlider_G">
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>B</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSlider" name="horizontalSlider_B">
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>显示格式</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_7">
      <item row="0" column="0">
       <widget class="QComboBox" name="com_time">
        <item>
         <property name="text">
          <string>hh:mm:ss</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>mm:ss</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ss</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>hh时mm分ss秒</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>mm分ss秒</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ss秒</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="com_date">
        <item>
         <property name="text">
          <string>yyyy-MM-dd</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MM-dd</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>dd</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>yyyy-MM-dd ddd</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>yyyy-MM-dd dddd</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>yyyy年MM月dd日 ddd</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>yyyy年MM月dd日 dddd</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MM月dd日 ddd</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MM月dd日 dddd</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
